poc-rails-template/serializer.mustache

```diff
@@ -0,0 +1,12 @@
+=begin
```

Comment:
Do we need begin and end?

Reply:
begin and end here enclose a section of multi-line comments which will print the info from the spec, like the description and version. I think we'll want it?